plugins/inputs/socketstat/README.md

```diff
@@ -0,0 +1,66 @@
+# SocketStat plugin
+
+The socketstat plugin gathers indicators from established connections, using iproute2's `ss` command.
+
+The `ss` command does not require specific privileges.
+
+The output format will have a very high cardinality. It should either be stored by an engine which doesn't suffer from it, of with a short retention policy.
```

Comment:
Could you please make the warning **bold** as otherwise people will miss it.

Reply:
done.